client-ci-analysis/ci-testfailure-analyzer/Models/AzDO/BuildInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text.Json.Serialization;
+
+namespace ci_testfailure_analyzer.Models.AzDO
```

Comment:
Why not using Azure DevOps official REST API client libarary?

https://www.nuget.org/packages/Microsoft.VisualStudio.Services.Client/

https://docs.microsoft.com/azure/devops/integrate/concepts/dotnet-client-libraries?view=azure-devops

Reply:
I didn't know it existed. Last time I asked about API, I got ref for test-[history](https://docs.microsoft.com/en-us/rest/api/azure/devops/test/test-history/query?view=azure-devops-rest-6.0) API, but it didn't return what I wanted. Anyway, I don't believe it's important for now. We can polish it with official REST API library later if needed.

Reply:
I'll check later if I can use for authentication, currently I'm using bearerToken from somewhere else, happy to replace it.